client/profile-wizard/steps/business-details/flows/selective-bundle/selective-extensions-bundle/index.js

```diff
@@ -191,21 +191,26 @@ export const ExtensionSection = ( {
 	);
 };
 
-export const createInstallExtensionOptions = ( extensions = [] ) => {
-	const defaultInstallExtensionOptions = { install_extensions: true };
-	return extensions.reduce( ( acc, curr ) => {
-		const plugins = curr.plugins.reduce(
-			( pluginAcc, { key, selected } ) => {
-				return { ...pluginAcc, [ key ]: selected ?? true };
```

Comment:
I removed the unnecessary `selected` check since it doesn't exist in the plugin object and API source.